nautobot_netbox_importer/tests/fixtures/nautobot_expectations.yaml

```diff
@@ -3842,9 +3861,23 @@
 #     termination_a_type: 11
 #     termination_b_id: 3
 #     termination_b_type: 3
-#     type: ''
+#     type: 'cat5e'
 #   model: dcim.cable
 
+#- fields:
```

Comment:
Y'know, I don't recall why the cable instances are commented out in this validation data. Is it because of the use of GenericForeignKeys making it hard to know in advance what the `termination_*_id` value will be?

Reply:
I'm not sure, and I felt like I've spent too much time on this PR already and didn't want to go down another rabbit hole